mne_bids/copyfiles.py

```diff
@@ -422,7 +422,7 @@ def copyfile_edf(src, dest, anonymize=None):
     # Ensure source & destination extensions are the same
     fname_src, ext_src = _parse_ext(src)
     fname_dest, ext_dest = _parse_ext(dest)
-    if ext_src != ext_dest:
+    if ext_src != ext_dest and ext_src != '.EDF':
```

Comment:
if the original extension is different from the new extension: raise an error

why the additional check to **only raise if the original extension is not EDF**?

if it makes sense (please tell me), why are you not also covering BDF?

I think it'd be nicer if we tweak copyfile to always write EDF files with lower-case `.edf`: Feed in `.EDF`, get a warning and a `.edf` :-)

here you want to prevent users from calling `copyfile_edf` on data with an EDF extension, right?

Reply:
Yeah makes sense. I did it the way you said.

Reply:
Lmk if this looks good?

Reply:
yes, that's better -thank you